share/metainfo/git-cola.appdata.xml

```diff
@@ -16,4 +16,8 @@
     </screenshot>
   </screenshots>
   <url type="homepage">https://git-cola.github.io/</url>
+  <releases>
+    <release version="3.12.0" date="2022-01-20" />
+  </releases>
+  <content_rating type="oars-1.1" />
```

Comment:
Thanks! This `<releases>` bit here is an extra step of maintenance that I'm going to be very prone to forget.

note to self: add a note about this to the `checklist` file in the `todo` branch.

Even better ~ update the `set-version` script in the `todo` branch to automatically append the new version here.

That script is run whenever we update the version ~ it goes and modifies the hard-coded version number that ends up needing to be written down in various files. I'll extended the script to handle this file as well before merging this PR and tagging the next release.

Thanks!